<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultifunctionalAddDlg</class>
 <widget class="QDialog" name="MultifunctionalAddDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultifunctionalAddDlg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_AddIC">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_ICName">
        <property name="text">
         <string>芯片名称</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_ICName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_ICManufacturer">
        <property name="text">
         <string>芯片厂家</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_ICManufacturer"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_ICRegCount">
        <property name="text">
         <string>总寄存器数</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_ICRegCount"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_ICEveryRegByteCount">
        <property name="text">
         <string>每个寄存器字节数</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_ICEveryRegByteCount"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_ICColorCount">
        <property name="text">
         <string>颜色个数</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_ICColorCount">
        <property name="inputMask">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="frame_OptBtn">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_Confirm">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Cancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
